pkg/reaper/nodereaper/nodereaper.go

```diff
@@ -533,6 +533,7 @@ func (ctx *ReaperContext) reapOldNodes(w ReaperAwsAuth) error {
 		// Skip if target node is self
 		if instance.NodeName == ctx.SelfNode {
 			log.Infof("self node termination attempted, skipping")
+			ctx.labelNode(instance.NodeName, "governor/status", "self-reap")
```

Comment:
Let's use the labels consistent with the existing ones, like `governor.keikoproj.io/foo`.

`self-reap` sounds like an action, too, as if it _will_ reap itself.

Perhaps

```
governor.keikoproj.io/status: current-node
```

or smth?